staging/src/k8s.io/apiserver/pkg/server/width/list.go

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I suggest moving this file to pkg/util/flowcontrol/

I'm not a huge fan of width directory, but I don't have a good suggestion, so maybe let's leave for now pkg/util/flowcontrol/width (or maybe pkg/util/flowcontrol/request/width.go ?)